errors.go

```diff
@@ -77,15 +77,6 @@ const (
 	ErrAuthtokExpired Error = C.PAM_AUTHTOK_EXPIRED
 	// ErrModuleUnknown indicates a module is not known.
 	ErrModuleUnknown Error = C.PAM_MODULE_UNKNOWN
-	// ErrBadItem indicates a bad item passed to pam_*_item().
-	ErrBadItem Error = C.PAM_BAD_ITEM
```

Comment:
This is actually defined in freebsd, so I feel having an `#ifdef` based check seems better